baybe/campaign.py

```diff
@@ -313,12 +315,18 @@ def recommend(
 
 
 def _add_version(dict_: dict) -> dict:
-    """Add the package version to the created dictionary."""
+    """Add the package version to the given dictionary."""
```

Comment:
Would you mind explaining these changes here one the version stuff? They are not related to `objective`, or am I missing something?

Reply:
yep, at leas a bulletpoint list laying put motivation for changes unrelated to the main PR theme is welcome as it avoids us making several threads asking why's in case somethingis not fully obvious (it also seems youve disallowed additional keywords for campaign below which is another unrelated change (not saying Im against that))

Reply:
Sure, sorry for being unclear here. Disallowing additional keywords for the objective was done while working on the PR because I noticed it's simply good/necessary to have it. I was testing for an error that I expected but didn't get because the keywords were still allowed. And that was when I noticed that some other hooks (Campaign, Recommender) also didn't have it, so just added it. (Is also very much related to that one issue you mentioned lateley, @Scienfitz). 

Disallowing the keywords for campaign required to drop the added version number because ... well ... it's an additional keyword :D Probably all this can be done more elegantly, but we'll come to it once we do mypy/hypothesis for campaigns.

And then lastly the highlighted change above ("created" -> "given") is just because "created" is wrong in this context (the dict comes in as a parameter to the function)

Reply:
Thanks for explaining. Still, as a general comment, we once explicitly agreed to NOT change things that are not related to a PR just because we noticed them. I personally would prefer if we also stick to this and then rather add comments like "Guys, btw, I noticed that X and Y which is problematic due to Z" and then fix in a follow-up PR

Reply:
Yes, policy approved :D but concerning the current PR, let me close the thread 👍🏼